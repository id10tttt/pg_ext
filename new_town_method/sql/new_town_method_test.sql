CREATE EXTENSION new_town_method;

select new_town_method(1, 2, 3, 4);
select new_town_method(1, 5, 3, 4);