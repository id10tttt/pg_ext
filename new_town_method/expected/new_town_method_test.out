CREATE EXTENSION new_town_method;
select new_town_method(1, 2, 3, 4);
 new_town_method  
------------------
 1.41421568627451
(1 row)

select new_town_method(1, 5, 3, 4);
 new_town_method  
------------------
 2.23809523809524
(1 row)

